package/lib/src/beam_page.dart

```diff
@@ -26,6 +26,7 @@ class BeamPage extends Page {
     this.popToNamed,
     this.type = BeamPageType.material,
     this.pageRouteBuilder,
+    this.routeBuilder,
```

Comment:
I'm think we should then remove `pageRouteBuilder` and use only `routeBuilder` for all. (`PageRoute` is a subclass of `Route`)

Reply:
oh... And when you remove `pagRouteBuilder`, you'll have to rename it to `routeBuilder` in all tests and examples where it's used (probably in no examples, but tests yes).